### PR TITLE
Create journal-week-2.md

### PR DESCRIPTION
**Pull request by boshnivolo**

### CODE REVIEW

**Comment by cmauban** [comment]
Great start!! I need to do better with translating english into JavaScript too. :+1:

**Comment by jmcreasman** [comment]
:+1: Keep swimming!

File: journal-week-2.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Week 3. First off: woo-hoo!
+
+This week has challenged my understanding of how to turn English into JavaScript. And while they don't seem to have that option on Google translate, I've been able to piece together the general structure of JavaScript syntax through Mozilla references, past assignments, and pure dumb luck.
```
Comment (kellymurray) [inline journal-week-2.md:3]: We should build a Google translate that's English -> JS and JS -> English

Comment (boshnivolo) [inline journal-week-2.md:3]: I think that should be our final project.

**Comment by jeremyrist** [comment]
You mentioned "at a lost" but I think you meant "at a loss"...but a big ol' :+1: for that tremendous cat gif and joke at the end!!

**Comment by karjac14** [comment]
:+1:  great Job with the rough draft! :smile_cat:

**Comment by cmauban** [comment]
Haha love the gif! :+1: Nice job explaining your struggles.. I feel you on that -_- for your next revision maybe connect one of the topics they gave us to what we have done. HAPPY BIRTHDAY! chug!

**Comment by jeremyrist** [comment]
Great job :+1: ! I think you're ready to go. If I was to give any suggestions I might start off by saying something like..."Can't believe I've made it to Week 3!" Or sumn like dat

File: journal-week-2.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+I've also noticed that we as a class have gotten better at working together on difficult assignments, and as a result we've been able to combine our relative knowledge of certain things into one big (sometimes) working code! This may be one of the most significant achievements we've unlocked in our first three weeks since it's not only applicable to our future careers, but our lives as well. This also seems to tie into our pair-programming where we work through similiar problems together in turns and see the varying results we reach.  
+
+And scoping in JavaScript seems to be like using sonar to figure out what should be there by seeing what bounces back...or something. So in that regard `var` and `function()` maps out what bounced back. And as the begin sending out more tests to search the deep depths of JavaScript for any sign of life, I get a better understanding of the strange functions and side-effects that inhabit my code. 
+
+<img src="https://media.giphy.com/media/seXDJPWScjklO/giphy.gif"/>
```
Comment (kellymurray) [inline journal-week-2.md:14]: yeeeees

**Pull request merged**